# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-radeon-printk.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.11/drivers/gpu/drm/radeon/radeon_display.c.vanilla	2024-10-12 12:02:35.811678760 +0200
+++ linux-6.11/drivers/gpu/drm/radeon/radeon_display.c	2024-10-12 12:08:54.495009640 +0200
@@ -777,8 +777,7 @@
 	DRM_INFO("Radeon Display Connectors\n");
 	list_for_each_entry(connector, &dev->mode_config.connector_list, head) {
 		radeon_connector = to_radeon_connector(connector);
-		DRM_INFO("Connector %d:\n", i);
-		DRM_INFO("  %s\n", connector->name);
+		DRM_INFO("Connector %d: %s\n", i, connector->name);
 		if (radeon_connector->hpd.hpd != RADEON_HPD_NONE)
 			DRM_INFO("  %s\n", hpd_names[radeon_connector->hpd.hpd]);
 		if (radeon_connector->ddc_bus) {
--- linux-6.11/drivers/gpu/drm/radeon/radeon_fbdev.c.vanilla	2024-10-12 12:04:48.241677883 +0200
+++ linux-6.11/drivers/gpu/drm/radeon/radeon_fbdev.c	2024-10-12 12:06:44.695010497 +0200
@@ -272,8 +272,7 @@
 	DRM_INFO("fb mappable at 0x%lX\n",  info->fix.smem_start);
 	DRM_INFO("vram apper at 0x%lX\n",  (unsigned long)rdev->mc.aper_base);
 	DRM_INFO("size %lu\n", (unsigned long)radeon_bo_size(rbo));
-	DRM_INFO("fb depth is %d\n", fb->format->depth);
-	DRM_INFO("   pitch is %d\n", fb->pitches[0]);
+	DRM_INFO("fb depth is %d, pitch is %d\n", fb->format->depth, fb->pitches[0]);
 
 	return 0;
 
